File: selenium/src/java/org/openqa/selenium/internal/ClassLookupStrategy.java
package org.openqa.selenium.internal;

import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.By;

public class ClassLookupStrategy implements LookupStrategy {
    public WebElement find(WebDriver driver, String use) {
        return driver.findElement(By.xpath("//*[@class='" + use + "']"));
    }
}

File: selenium/src/csharp/WebdriverBackedSelenium/Internal/SeleniumEmulation/LookupStrategies/ClassLookupStrategy.cs
using System;
using OpenQA.Selenium;

namespace Selenium
{
    internal class ClassLookupStrategy : ILookupStrategy
    {
        public IWebElement Find(OpenQA.Selenium.IWebDriver driver, string use)
        {
            return driver.FindElement(By.ClassName(use));
        }
    }
}
